csaf_2.1/prose/edit/src/distributing.md

```diff
@@ -162,8 +162,13 @@ The use of the scheme "HTTPS" is required. See [cite](#RFC8615) for more details
 
 ### Requirement 10: DNS path
 
-The DNS record `csaf.data.security.domain.tld` SHALL resolve as a web server which serves directly
-the `provider-metadata.json` according to requirement 7. That implies that redirects SHALL NOT be used.
+Assuming that the organization's main domain is `domain.tld`, the DNS record `csaf.data.security.domain.tld` SHALL resolve
+as a web server which serves directly the `provider-metadata.json` according to requirement 7.
```

Comment:
Nit: Maybe more specifically instead of:

> as a web server which serves directly the

write something like:

> to the IP address of a web server directly serving the

(rationale: DNS resolves to IP addresses not to "web servers")

Reply:
Done in [7a9fae0](https://github.com/oasis-tcs/csaf/pull/868/commits/7a9fae02692ae1c040083ee8c98e7ab3ec36b999)